docs/user_guide/SIMP_Administration/General_Administration.rst

```diff
@@ -0,0 +1,292 @@
+General Administration
+======================
+
+.. WARNING::
+
+    While working with the system, keep in mind that Puppet does not work well
+    with capital letters in host names. Therefore, they should not be used.
+
+The SIMP Environment
+--------------------
+
+SIMP fully supports `Puppet Environments`_ and, by default, installs into an
+environment named ``simp``. This environment is symlinked to the ``production``
+environment by ``simp config`` but that symlink will **not** be overwritten on
+update so you may freely change or replace the symlink to meet your needs.
+
+There are a couple of paths on the system that are environment related.
+
+/var/simp
+^^^^^^^^^
+
+This space holds all static, non-Puppet created files. It is generally used for
```

Comment:
@lnemsick-simp Those are static, non-Puppet created files. Sure, they might change, but it's not the result of any manual system.